docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
     networks:
       - nhsd_bcnetwork
     environment:
+      - ASPNETCORE_ENVIRONMENT=Development

```

Comment:
Will we need to make any changes to the helm charts related to this alteration?

Reply:
I have no idea as I no nothing about helm. Can @assoomro or @stuart-bass-bjss help?

Reply:
We'll need to add that env. var to the helm chart. I suggest we raise a task for this to keep track of it.

Reply:
I've added a platform [task (5521)](https://dev.azure.com/BuyingCatalog/Buying%20Catalogue/_workitems/edit/5521) to the story. Thanks @assoomro.